be/src/vec/olap/vgeneric_iterators.h

```diff
@@ -99,7 +99,7 @@ class VMergeIteratorContext {
     VMergeIteratorContext& operator=(const VMergeIteratorContext&) = delete;
     VMergeIteratorContext& operator=(VMergeIteratorContext&&) = delete;
 
-    ~VMergeIteratorContext() {}
+    ~VMergeIteratorContext();
```

Comment:
`~VMergeIterator() ` why not ` return_ iterator`

Reply:
returned in `VMergeIteratorContext::~VMergeIteratorContext` which is it's imp